src/components/AccessControl/AccessControl.vue

```diff
@@ -18,23 +19,40 @@ export default {
       userRoleNum: -1,
     };
   },
-  created() {
-    this.userRoleNum = userState.getUserAdminLevel();
+  watch: {
+    adminLevel: {
+      immediate: true,
+      handler() {
+        this.userRoleNum = userState.getUserAdminLevel();
+
+        if (this.adminLevel !== this.userRoleNum) {
+          this.setUser();
+        }
+      },
+    },
   },
   computed: {
+    ...mapState('user', {
+      adminLevel: 'adminLevel',
+    }),
     userRole() {
       switch (this.userRoleNum) {
         case 0:
           return 'user';
         case 2:
           return 'admin';
         default:
-          return 'error';
+          return 'guest';
```

Comment:
Confused what you're doing here, what about "participating families"?

Reply:
I didn't have an account that had pf, that needs to be added as case 1: return 'pf'